SpaceX/RocketsScreen/SwiftUIRockets/Rocket/RocketView.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  RocketView.swift
+//  SpaceX
+//
+//  Created by Julia on 11.09.2022.
+//
+
+import SwiftUI
+
+struct RocketView: View {
+    var nameSpace: Namespace.ID
```

Comment:
private

Reply:
We can't make it private, because we have to pass id into it from another view.